x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -585,7 +585,8 @@ export const AlertForm = ({
         position="right"
         type="questionInCircle"
         content={i18n.translate('xpack.triggersActionsUI.sections.alertForm.checkWithTooltip', {
-          defaultMessage: 'Define how often to evaluate the condition.',
+          defaultMessage:
+            'Define how often to evaluate the condition. Checks may be queued and run at an increased interval.',
```

Comment:
I don't think I'd understand what "increased interval" means, to be honest.
My thinking is that the goal here is to communicate that we will do our best effort to run at this interval, but at times this might increase due to resource contention.

Perhaps we can try something like:
```suggestion
            'Define how often to evaluate the condition. Checks are queued and run as near to this interval as capacity allows.',
```

I feel like the wording isn't great, but "an increased interval" suggests that subsequent executions may be further delays as well, which they shouldn't be.